Beta_Integrations/Telegram/CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 ## [Unreleased]
+-
 Updated the integration's image.
```

Comment:
something is wrong with the release notes here

Reply:
What is it?

Reply:
move `Updated the integration's image.` to not be under `Unreleased`